great_expectations/checkpoint/checkpoint.py

```diff
@@ -152,6 +150,9 @@ def run(
         expectation_parameters: Dict[str, Any] | None = None,
         run_id: RunIdentifier | None = None,
     ) -> CheckpointResult:
+        if not self.validation_definitions:
+            raise CheckpointRunWithoutValidationDefinitionError()
```

Comment:
fyi in v0.18, we raise this: `reat_expectations.exceptions.exceptions.CheckpointError: Checkpoint "<CHECKPOINT_NAME>" must be called with a validator or contain either a batch_request or validations.`